SPECS/shim/shim.spec

```diff
@@ -0,0 +1,33 @@
+%global debug_package %{nil}
+%global release_number 1
+Summary:        First stage UEFI bootloader
+Name:           shim
+Version:        15.4
+Release:        %{release_number}%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/rhboot/shim
+# This signed-shim tarball contains the shim binary signed with
+# the Microsoft UEFI CA key
+Source0:        signed-%{name}-%{version}-%{release_number}.tar.gz
+ExclusiveArch:  x86_64
+
+%description
+Initial UEFI bootloader that handles chaining to a trusted full bootloader
+under secure boot environments.
+
+%prep
+%autosetup
+
+%install
+install -d %{buildroot}/boot/efi/EFI/BOOT
+cp shimx64.efi %{buildroot}/boot/efi/EFI/BOOT/bootx64.efi
```

Comment:
Nit: since we're already using `install` to create a directory, we could do that to move the shim EFI as well for the sake of consistency. In addition to that we'd also be able to use the `-m` option to explicitly call out the access rights to the file.

Reply:
Fixed